log_stdout.module

```diff
@@ -1,23 +1,42 @@
 <?php
 
+function log_level() {
+  $severity = getenv('PENGUIN_LOG_LEVEL');
+
+  if ($severity) {
+    return defined($severity) ? constant($severity) : false;
```

Comment:
How does this work if PENGUIN_LOG_LEVEL is CRITICAL (which is 0)?  Doesn't that evaluate to false below?